frontend/www/js/omegaup/components/course/Form.vue

```diff
@@ -287,6 +294,7 @@ export default class CourseDetails extends Vue {
   @Prop({ default: '' }) invalidParameterName!: string;
   @Prop() allLanguages!: string[];
   @Prop() searchResultSchools!: types.SchoolListItem[];
+  @Prop() hasVisitedSection!: boolean;
```

Comment:
Creo que ya vi a que se debe el error de las pruebas de Cypress. 

Resulta que el componente donde estás mostrando el introJS se utiliza para 2 cosas:

- Para crear un curso
- Para editar un curso

Entonces, parece que el modal se está mostrando también cuando se edita un curso.

La forma más sencilla que se me ocurre para arreglar la falla es agregarle un valor por defecto a esta prop:
```suggestion
  @Prop({ default: true }) hasVisitedSection!: boolean;
```
si lo configuramos como `true`, en los lugares donde no lo envíes explícitamente no mostrará el modal, en este caso en la vista de "Editar curso"